rapids/test_rapids.py

```diff
@@ -27,39 +28,38 @@ def _run_dask_test_script(self, name):
             self.DASK_TEST_SCRIPT_FILE_NAME)
         self.assert_instance_command(name, verify_cmd)
 
-    def verify_spark_instance(self, name):
+    def verify_spark3_instance(self, name):
+        self.assert_dataproc_job(
+            self.name, "pyspark", "{}/rapids/{}".format(self.INIT_ACTIONS_REPO,
+                                                        self.SPARK_TEST_SCRIPT_FILE_NAME))
+
+    def verify_spark2_instance(self, name):
         self.assert_instance_command(name, "nvidia-smi")
 
-    @parameterized.parameters(("STANDARD", ["m"], GPU_P100, False),
-                              ("STANDARD", ["m"], GPU_P100, True))
-    def test_rapids_dask(self, configuration, machine_suffixes, accelerator,
-                         dask_cuda_worker_on_master):
+    @parameterized.parameters(("STANDARD", ["m", "w-0"], GPU_P100))
+    def test_rapids_dask(self, configuration, machine_suffixes, accelerator):
         # Init action supported on Dataproc 1.3+
         if self.getImageVersion() < pkg_resources.parse_version("1.3"):
             return
 
         metadata = 'gpu-driver-provider=NVIDIA,rapids-runtime=DASK'
-        if dask_cuda_worker_on_master:
-            master_accelerator = accelerator
-        else:
-            metadata += ',dask-cuda-worker-on-master=false'
-            master_accelerator = None
+        master_accelerator = accelerator
         self.createCluster(configuration,
                            self.INIT_ACTIONS,
                            metadata=metadata,
                            master_accelerator=master_accelerator,
                            worker_accelerator=accelerator,
                            optional_components=['ANACONDA'],
                            machine_type='n1-standard-2',
-                           timeout_in_minutes=20)
+                           timeout_in_minutes=30)
```

Comment:
This still wasn't enough time for my tests, perhaps raise to 60?

Reply:
yeah, I would say 60 minutes.

Reply:
We can also drop 1.x test since it is not an easy experience to set up.

Reply:
@medb what do you think? Dask setup on Dataproc 1.x takes ~hour and is much better optimized for Dataproc 2.0. I think dropping the test here is ok.

Reply:
Agree, this makes sense. 1.5 seems to work so I added it back.